Tests/StreamChatTests/Repositories/MessageRepository_Tests.swift

```diff
@@ -123,6 +123,27 @@ final class MessageRepositoryTests: XCTestCase {
         XCTAssertNotNil(result?.value)
     }
 
+    func test_sendMessage_skipPush() throws {
+        let id = MessageId.unique
+        try createMessage(id: id, localState: .pendingSend, skipPush: true)
+        let expectation = self.expectation(description: "Send Message completes")
+        repository.sendMessage(with: id) { _ in
+            expectation.fulfill()
+        }
+
+        wait(for: [apiClient.request_expectation], timeout: defaultTimeout)
```

Comment:
Why not just use a single `wait(for:)` with both expectations?

Reply:
I'm actually not sure, I just followed the other tests here, and the way our APIClient is right now it is a bit hard to test so I assumed this weird stuff was needed 😆